okhttp/src/commonJvmAndroid/kotlin/okhttp3/Request.kt

```diff
@@ -423,4 +423,59 @@ class Request internal constructor(
 
     open fun build(): Request = Request(this)
   }
+
+  /**
+   * Returns a cURL command equivalent to this request, useful for debugging and reproducing requests.
+   *
+   * This includes the HTTP method, headers, request body (if present), and URL.
+   *
+   * Example:
+   * ```
+   * curl -X POST -H "Authorization: Bearer token" --data "{\"key\":\"value\"}" "https://example.com/api"
+   * ```
+   *
+   * @return a cURL command string representing this request.
+   */
+  fun curl(): String {
```

Comment:
This should probably be named `toCurl`, since it's not a property of the request itself.